spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/data/parquet/vectorized/TestParquetVectorizedReads.java

```diff
@@ -161,28 +164,34 @@ void assertRecordsMatch(
       boolean reuseContainers,
       int batchSize)
       throws IOException {
-    Parquet.ReadBuilder readBuilder =
-        Parquet.read(inputFile)
-            .project(schema)
-            .recordsPerBatch(batchSize)
-            .createBatchedReaderFunc(
-                type ->
-                    VectorizedSparkParquetReaders.buildReader(
-                        schema, type, Maps.newHashMap(), null));
-    if (reuseContainers) {
-      readBuilder.reuseContainers();
-    }
-    try (CloseableIterable<ColumnarBatch> batchReader = readBuilder.build()) {
-      Iterator<GenericData.Record> expectedIter = expected.iterator();
-      Iterator<ColumnarBatch> batches = batchReader.iterator();
-      int numRowsRead = 0;
-      while (batches.hasNext()) {
-        ColumnarBatch batch = batches.next();
-        numRowsRead += batch.numRows();
-        TestHelpers.assertEqualsBatch(schema.asStruct(), expectedIter, batch);
-      }
-      assertThat(numRowsRead).isEqualTo(expectedSize);
-    }
+    assertNoLeak(
+        "testRecordsMatch",
```

Comment:
don't we need to pass in the test method name here?

Reply:
ah yeah, we don't have it at this point in the call, originally I was just using the file name of what's being read ... Let me see if I can get it from JUNIT somehow

Reply:
ah I see this got screwed up when I was moving code between the spark modules.